src/odemis/driver/hamamatsurx.py

```diff
@@ -1122,6 +1122,10 @@ def readCommandResponse(self):
                     # when socket timed out (receiving no response)
                     logging.debug("Timeout on the socket, will wait for more data packets.")
                     continue
+                if not returnValue:
```

Comment:
Maybe add a comment why we do this trick and a TODO to actually follow up on this issue.